graphene_django_plus/mutations.py

```diff
@@ -122,17 +122,20 @@ def _get_fields(model, only_fields, exclude_fields, required_fields, registry):
 
         if required_fields is not None:
             required = name in required_fields
-            f.kwargs["required"] = required
+            if hasattr(f, 'kwargs'):
+                f.kwargs["required"] = required
         else:
             if isinstance(field, (ManyToOneRel, ManyToManyRel)):
                 required = not field.null
             else:
                 required = not field.blank and field.default is NOT_PROVIDED
 
-            f.kwargs["required"] = required
+            if hasattr(f, 'kwargs'):
+                f.kwargs["required"] = required
 
         s = schema_for_field(field, name, registry)
-        required = f.kwargs["required"]
+        if hasattr(f, 'kwargs'):
```

Comment:
consistency would be nice for single-double quotes. As discussed - we might add a package to verify that, but for now - let's not introduce anything additional to keep this PR as small as possible and later on - try to create a PR in the upstream